src/sass/checkboxes/_base.scss

```diff
@@ -3,121 +3,103 @@
 
 @use '../core' as *;
 
-.#{$prefix}-checkbox {
-  position: absolute;
-  left: -$spacing-sm;
-  opacity: 0;
-
-  & + .#{$prefix}-label,
-  .#{$prefix}-checkbox-wrapper & ~ .#{$prefix}-label {
-    font-size: $spacing-sm;
-
-    /**
-     * Helps visually align checkbox lists
-     */
-    margin-left: .2rem;
-    font-weight: $font-weight-regular;
-
-    &::before {
-      display: inline-block;
-      content: ' ';
-      border-radius: .125rem;
-      margin-right: .75rem;
-      width: $spacing-sm;
-      height: $spacing-sm;
-      transition: all .2s ease;
-
-      /**
-       * There are some magic numbers here that help improve rendering
-       * of box-shadows in Firefox and IE. Making them just a smidge
-       * wider (.08rem instead of .0625rem/1px) than one pixel compensates
-       * for some weird aliasing that is happening. As a bonus it seem to
-       *  make radios and checkboxes look optically closer the 1px border
-       * used on other inputs like text inputs, textareas, and selects.
-       */
-      box-shadow: 0 0 0 .08rem $color-black-400;
-
-      /**
-       * This helps visually center the label with the pseudo
-       * element checkbox.
-       */
-      vertical-align: middle;
-      margin-top: -.1rem;
-    }
+.rvt-checkbox {
+  display: inline-block;
+  padding-left: $spacing-lg;
+  position: relative;
+
+  input[type=checkbox] {
+    cursor: pointer;
+    height: $spacing-md;
+    left: 0;
+    margin: 0;
+    top: 0;
+    width: $spacing-md;
```

Comment:
Now I remember another reason I added left padding on the label. It ensured that the clickable hit area extended fully from the visual checkbox through to the end of the label. Now, there's a gap. But this can be overcome by adjusting the checkbox width. This technique works in Chrome, Firefox, and Safari.

```css
.rvt-checkbox input[type=checkbox] {
  width: 2rem;
}

.rvt-checkbox--sr-only-label input[type=checkbox] {
  width: 1.5rem;
}
```

Despite needing this additional change, I still like doing this, rather than having padding on the label and description.